acceptance/tests/mesh-gateway/mesh_gateway_test.go

```diff
@@ -112,9 +115,9 @@ func TestMeshGatewayDefault(t *testing.T) {
 	// gateways.
 	logger.Log(t, "creating proxy-defaults config")
 	kustomizeDir := "../fixtures/bases/mesh-gateway"
-	k8s.KubectlApplyK(t, primaryContext.KubectlOptions(t), kustomizeDir)
+	k8s.KubectlApplyK(t, secondaryContext.KubectlOptions(t), kustomizeDir)
 	helpers.Cleanup(t, cfg.NoCleanupOnFailure, func() {
-		k8s.KubectlDeleteK(t, primaryContext.KubectlOptions(t), kustomizeDir)
+		k8s.KubectlDeleteK(t, secondaryContext.KubectlOptions(t), kustomizeDir)
 	})
```

Comment:
should we keep this test using the primary for more coverage? I think the main one we care about is the secure test anyways